hash/saturn.xml

```diff
@@ -27535,6 +27663,23 @@ Sega logo animation draws offset [VDP2]
 		</part>
 	</software>
 
+	<!-- Version 1.000, graphic glitches with 32mb RAM cart, build date 19960804 -->
+	<software name="rbffa" supported="no" cloneof="rbff">
+		<description>Real Bout Garou Densetsu (Japan, Sample)</description>
```

Comment:
`cloneof` should be before `supported` for consistency with everything else in hash folder.